xskillscore/core/probabilistic.py

```diff
@@ -1128,21 +1117,15 @@ def roc(
         ``true positive rate`` and ``false positive rate`` contain
         ``probability_bin`` dimension with ascending ``bin_edges`` as coordinates.
 
-
     Examples
     --------
-    >>> f = xr.DataArray(
-    ...     np.random.normal(size=(1000)),
-            coords=[('time', np.arange(1000))]
-    ... )
-    >>> o = xr.DataArray(
-    ...    np.random.normal(size=(1000)),
-    ...    coords=[('time', np.arange(1000))]
-    ... )
+    >>> f = xr.DataArray(np.random.normal(size=(1000)),
+    ...                  coords=[('time', np.arange(1000))])
+    >>> o = f.copy()
```

Comment:
Why copy? Do you want to get perfect score?

Reply:
yeah it simplifies writing the example and also shows the metric is bounded at 1 for a prefect forecast.

The current example is a bit dull with two random datasets (https://github.com/xarray-contrib/xskillscore/blob/main/xskillscore/core/probabilistic.py#L1145) and shows it's < 0.5